test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -672,4 +662,28 @@ public void testClobColumnAsMetadata() throws Exception {
     expected.add("col2", content);
     assertEquals(expected, response.getMetadata());
   }
+
+  @Test
+  public void testClobColumnAsMetadata() throws Exception {
+    String content = "Hello World";
+    executeUpdate("create table data(id int, content clob)");
```

Comment:
If NCLOB does indeed come back to us as CLOB, add a comment for this method to that effect. Same for LONGVARCHAR and the test below. Please add a test for INTEGER as well, just for completeness now that we can.